src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.cs

```diff
@@ -2545,6 +2545,293 @@ public new abstract class Arm64 : AdvSimd.Arm64
         public static unsafe Vector<ulong> OrAcross(Vector<ulong> value) => OrAcross(value);
 
 
+        /// Saturating decrement by number of halfword elements
+
+        /// <summary>
+        /// int32_t svqdech_pat[_n_s32](int32_t op, enum svpattern pattern, uint64_t imm_factor)
+        ///   SQDECH Xtied, Wtied, pattern, MUL #imm_factor
+        /// </summary>
+        public static unsafe int SaturatingDecrementBy16BitElementCount(int value, [ConstantExpected(Min = 1, Max = (byte)(16))] byte scale, [ConstantExpected] SveMaskPattern pattern = SveMaskPattern.All) => SubtractSaturateScalar(value, (int)(scale * decodePredicateCount(Unsafe.SizeOf<Vector<short>>() / sizeof(short), pattern)));
```

Comment:
remind me, where does the decision happen whether to JIT the method as intrinsic or use fallback?

Reply:
If `impHWIntrinsic()` returns `nullptr` then use fallback. Check the calls to `CheckHWIntrinsicImmRange()` within `impHWIntrinsic()`

Reply:
ah yes. I was looking for `MaybeNoJmpTableImm()` instead of `NoJmpTableImm`.